Jenkinsfile

```diff
@@ -31,37 +61,52 @@ pipeline {
                 sh "zip -r release.zip *"
                 archiveArtifacts "release.zip"
             }
+            post {
+                always {
+                    cleanWs()
+                }
+            }
         }
         stage('Composer') {
             agent {
                 docker {
                     image 'composer:2.0.11'
-                    args '--mount type=volume,source=composer-cache,destination=/root/.composer/cache '
+                    args '--mount type=volume,source=composer-cache2,destination=/root/.composer/cache '
```

Comment:
I seem to have corrupted the volume `composer-cache`, perhaps docker needs a restart